package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@mobilestack-xyz/mobilestack",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "20.17.0"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e",
```

Comment:
Uses a fixed yarn version so CI and local setups behave the same.

Otherwise I'd get different `yarn.lock` diffs between my local setup and GH Actions.